README.md

```diff
@@ -48,3 +48,4 @@ Search links that point directly to suitable issues to contribute to on GitHub
 - [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) - GitHub talk on how to make a pull request
 - [GitHub Learning Resources](https://help.github.com/articles/git-and-github-learning-resources/) - Git and GitHub learning resources
 - [Pro Git](https://git-scm.com/book/en/v2) - The entire Pro Git book, written by Scott Chacon and Ben Straub and published by Apress.
+- [45 Github Issues Dos and Don’ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612)- Do's and Don't on      GitHub
```

Comment:
Although this article is on GitHub and GitHub is a site to perform version control, the contents of that article I feel are more general than how to use version control. The article is how to act on GitHub as an open-source community. So I think this better belongs in the "Contributing to open source in general" section. 

Additionally, I've merged in another pull request that conflicts with your changes. So if you can move your addition to the first section on contributing to open source in general, I can go ahead and merge your changes. Let me know if you have any questions or need help making these changes. Thanks!